packages/flutter/lib/src/material/tab_bar_theme.dart

```diff
@@ -0,0 +1,94 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'tabs.dart';
+
+/// Applies a theme to descendant [TabBar] widgets.
```

Comment:
This class doesn't apply a theme (that would be an inherited widget), it just defines one.

Reply:
Done.